dbms/src/Storages/Transaction/TiDB.cpp

```diff
@@ -110,14 +110,22 @@ Field ColumnInfo::defaultValueToField() const
     }
     switch (tp)
     {
-    // TODO: Consider unsigned?
```

Comment:
Can this function handle `create table t ( a int default 18446744073709551616)`? (18446744073709551616==MAX(Int64) + 1) Or will that be blocked by TiDB?

Reply:
It will be blocked by TiDB. And
```
mysql> create table t ( a int unsigned default 4294967295);
Query OK, 0 rows affected (0.07 sec)
```
this is OK. `4294967295 == MAX(UInt32)`. TiDB distinguishes `int unsigned` from `int` by `flag` rather than `tp`. Therefore, we do not need to consider unsigned type here.

Reply:
add comment about the unsigned default value to the code pls

Reply:
done